server/loader.go

```diff
@@ -128,7 +128,7 @@ func FromConfiguration(data *api.ServerConfigurationResponse) (*Server, error) {
 
 	// If the server's data directory exists, force disk usage calculation.
 	if _, err := os.Stat(s.Filesystem.Path()); err == nil {
-		go s.Filesystem.HasSpaceAvailable()
+		go s.Filesystem.HasSpaceAvailable(true)
```

Comment:
This doesn't need to be `true`, it is "forcing" calculation simply so there is a value returned, it doesn't need to be the most accurate value. In this case, we can pass false and not even put this in a go-routine.